mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVMPass.cpp

```diff
@@ -69,6 +70,7 @@ void ConvertSYCLToLLVMPass::runOnOperation() {
 
   ConversionTarget target(*context);
   target.addIllegalDialect<sycl::SYCLDialect>();
+  target.addLegalOp<sycl::SYCLLocalIDOp>();
```

Comment:
Please add a TODO comment to indicate that this is temporary.

Reply:
Done!